CRITICAL - 2020-11-22 11:05:13 --> syntax error, unexpected '$data' (T_VARIABLE)
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Router\Router.php(195): CodeIgniter\Router\Router->autoRoute('auth/register')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(793): CodeIgniter\Router\Router->handle('auth/register')
#2 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(364): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-22 11:05:13 --> syntax error, unexpected '$data' (T_VARIABLE)
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Router\Router.php(195): CodeIgniter\Router\Router->autoRoute('auth/register')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(793): CodeIgniter\Router\Router->handle('auth/register')
#2 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(364): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-22 11:08:39 --> Undefined variable: userModel
#0 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(12): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 12, Array)
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#2 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-22 11:08:52 --> Undefined variable: userModel
#0 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(12): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 12, Array)
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#2 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-22 11:09:10 --> Undefined variable: userModel
#0 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(13): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 13, Array)
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(890): App\Controllers\Auth->__construct()
#2 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-22 11:48:48 --> syntax error, unexpected 'session' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST)
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(7): spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Router\Router.php(583): include_once('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Router\Router.php(195): CodeIgniter\Router\Router->autoRoute('auth/index')
#6 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(793): CodeIgniter\Router\Router->handle('auth/index')
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(364): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#8 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2020-11-22 11:52:16 --> Undefined variable: validation
#0 C:\xampp\htdocs\ci4template\app\Views\auth\login.php(25): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 25, Array)
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/login', Array, true)
#3 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(27): view('auth/login', Array)
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-22 11:52:21 --> Undefined variable: validation
#0 C:\xampp\htdocs\ci4template\app\Views\auth\login.php(25): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 25, Array)
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/login', Array, true)
#3 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(27): view('auth/login', Array)
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-22 11:52:24 --> Undefined variable: validation
#0 C:\xampp\htdocs\ci4template\app\Views\auth\login.php(25): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 25, Array)
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/login', Array, true)
#3 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(27): view('auth/login', Array)
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-22 11:54:03 --> Undefined variable: validation
#0 C:\xampp\htdocs\ci4template\app\Views\auth\login.php(25): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 25, Array)
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/login', Array, true)
#3 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(27): view('auth/login', Array)
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#9 {main}
CRITICAL - 2020-11-22 12:00:22 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(35): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:01:33 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(38): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:01:54 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(40): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:02:25 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:02:27 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:02:34 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:02:36 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:02:39 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:02:48 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(40): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:02:51 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:02:55 --> Invalid file: /auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('/auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('/auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('/auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:02:58 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:03:41 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:03:54 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:03:56 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:03:58 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:04:03 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:04:13 --> Invalid file: auth/index.php
#0 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('auth/index.php')
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Common.php(1119): CodeIgniter\View\View->render('auth/index', Array, true)
#2 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(39): view('auth/index', Array)
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#4 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2020-11-22 12:07:13 --> Undefined variable: validation
#0 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(20): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 20, Array)
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#2 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2020-11-22 12:08:07 --> Undefined variable: validation
#0 C:\xampp\htdocs\ci4template\app\Controllers\Auth.php(20): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 20, Array)
#1 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(918): App\Controllers\Auth->index()
#2 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci4template\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\ci4template\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(34): require_once('C:\\xampp\\htdocs...')
#6 {main}
